### PR TITLE
Update notifications and info boxes styles for accessibility

### LINKED ISSUE #20691: Accessible designs for notifications & inline messages
Problem: (some of) the current notifications / banners / inline messages aren’t WCAG compliant 
Solution: create & implement on-brand WCAG compliant designs for each of the elements

Actions:
1. List all relevant elements (done)
2. Create new designs (done)
3. Implement designs

Component: Notifications
1. Informative
2. Success
3. Warning
4. Error (alert)

Analysis
There are four types of notifications (as mentioned above) that are commonly used across products, the fonts used in them should have enough contrast (4.5:1 minimum requirement) to comply with accessibility standards and same goes with UI elements (3:1).

We currently have these notifications but they don’t fully comply with accessibility and have different variations of these which are not being used consistently across the product, some can be found on github and some are not there.

Proposed solution
Notifications:
- Notifications are unified = consistency → we don’t have two different versions for each type
- Colours are now accessible
- black font
- light shade of colour in the background 
- icon and border comply with accessibility standards for UI element 
- if notification has a subject, it goes in bold → hierarchy

Inline messages:
- Colours are now accessible
- Dark grey font
- light shade of grey in the background 

image
image
image
image
image
image
image
image
image
image
image

### PR DESCRIPTION
**Pull request by bx80**
### Description:

Fixes #20691

The PR updates the styling of notifications and info boxes throughout the application to improve accessibility.

For some reason we have components for both notifications and alerts, each of which can be of the types: success, info, warning or error. The only real difference I can see is that notifications can optionally have a close button. There seems to be roughly equal usage of both components throughout the application. If there is no other difference then we might want consider deprecating alerts at some point in favor of notifications.

I’ve applied the new styles to both notifications and alerts and also added some notification examples to the UI demo screen so that they can be covered by UI tests.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by michalkleiner** [comment]
Created new ticket to look into the consolidation of the components — https://github.com/matomo-org/matomo/issues/21079.

**Comment by sgiehl** [comment]
@bx80 for such intense changes it's good to temporarily update the submodules to the correlating branches in the PR. That way you can directly see if all tests of the submodule plugins are passing

File: plugins/Login/tests/UI/expected-screenshots/Login_bruteforcelog_noentries.png
Comment (sgiehl) [inline plugins/Login/tests/UI/expected-screenshots/Login_bruteforcelog_noentries.png:1]: Looks like the icon for the platform menu got lost.

File: plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_notification_plugincheck_exceededLicense.png
Comment (sgiehl) [inline plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_notification_plugincheck_exceededLicense.png:1]: Should links be blue in those notifications?

File: plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_paid_plugin_details_exceeded_license_superuser.png
Comment (sgiehl) [inline plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_paid_plugin_details_exceeded_license_superuser.png:1]: Guess we missed that when changing the links, but guess those for authors and website should be blue, too.

File: plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_superuser_enable_plugins_admin_with_multiserver_enabled.png
Comment (sgiehl) [inline plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_superuser_enable_plugins_admin_with_multiserver_enabled.png:1]: The more links could be blue here.

File: plugins/Morpheus/Controller.php [hunk 1]
```diff
@@ -96,13 +97,53 @@ public function demo()
...
+    </Notification>
+    <Notification context="warning" :noclear="true">
+      <strong>Warning</strong>:
+      Better <a>check</a> yourself, you\'re not looking too good. There is no close button.
+    </Notification>
+    <Notification context="error" :noclear="true">
+      <strong>Error</strong><br>
+      Change <a>a few things</a> and try submitting again, this notification does not have a close button, but it does contain multiple lines of text. Textus sine sensu ultra finem lineae pergit ante movens infra et iterum ad sinistram.
+    </Notification>',
+            'desc' => 'This is a Vue component. You can use the :noclear="true" property to hide the close button',
```
Comment (sgiehl) [inline plugins/Morpheus/Controller.php:140]: So is the last remaining difference between Alert and Notification, that Notifications by default are "clearable"?
Or is there anything else? If not we could maybe consider to deprecate/remove one of them in the future.

File: plugins/Overlay/tests/UI/expected-screenshots/Overlay_loaded_from_actions.png
Comment (sgiehl) [inline plugins/Overlay/tests/UI/expected-screenshots/Overlay_loaded_from_actions.png:1]: The notification here looks cut off. We should check the reason maybe.

File: plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_anonymizelogdata_anonymizeip_and_visit_column_cancelled.png
Comment (sgiehl) [inline plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_anonymizelogdata_anonymizeip_and_visit_column_cancelled.png:1]: Shouldn't all drop downs use the same arrow? Our customized ones now have another arrow then the default drop downs rendered by materialize. We can go with that for sure, it just looks a bit inconsistent maybe.

File: plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_users_opt_out_default.png
Comment (sgiehl) [inline plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_users_opt_out_default.png:1]: The white font on the light blue background might be a bit hard to read 🙈

File: plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated.png
Comment (sgiehl) [inline plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated.png:1]: That one doesn't have the correct result anymore. It should show the new name of the segment. Maybe the screenshot needs another update.

File: plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated_details.png
Comment (sgiehl) [inline plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated_details.png:1]: Needs an update as well, as the segment name isn't correct anymore.

File: plugins/TwoFactorAuth/tests/UI/expected-screenshots/TwoFactorAuthUsersManager_edit_with_2fa_reset_confirmed.png
Comment (sgiehl) [inline plugins/TwoFactorAuth/tests/UI/expected-screenshots/TwoFactorAuthUsersManager_edit_with_2fa_reset_confirmed.png:1]: The back link here uses `icon-arrow-left-2`, which now looks almost the same as the new `icon-arrow-left`. Maybe we should remove the second one, as it's only used in the usersmanage anyway.

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png
Comment (sgiehl) [inline plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png:1]: The tooltip is missing, which should be part of the UI tests. If the test is broken in general, we should create a new issue to fix it. But we should not update the screenshot here in case it doesn't have the correct result.

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_bounce_rate.png
Comment (sgiehl) [inline plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_bounce_rate.png:1]: Same here, tooltip is missing.

File: plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_permissions_bulk_access_set.png
Comment (sgiehl) [inline plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_permissions_bulk_access_set.png:1]: That update is also incorrect. The back link is missing. Guess this was one of the random failures...

File: plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_tab.png
Comment (sgiehl) [inline plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_tab.png:1]: Note: The new icon might look a bit less important in this case than before.

File: tests/UI/expected-screenshots/UIIntegrationTest_actions_pages_tooltip_help.png
Comment (sgiehl) [inline tests/UI/expected-screenshots/UIIntegrationTest_actions_pages_tooltip_help.png:1]: Should we maybe change this help box to look the same as an info notification?

File: tests/UI/expected-screenshots/UIIntegrationTest_category_help.png
Comment (sgiehl) [inline tests/UI/expected-screenshots/UIIntegrationTest_category_help.png:1]: This doesn't look better then before. Maybe we are missing some definitions how that help notification should look like?

**Review by sgiehl — CHANGES_REQUESTED** [review]
Had a look through all the changed screenshots. Left a couple of comments, where additional changes might be required.
Some of them might be unrelated to the purpose of this PR and could also be changed independently. In some cases the resulting screenshots aren't correct anymore and need further updates.

In general we should maybe discuss if we want to deprecate either the Notification or the Alert component. It seems they already look the same, so their is no benefit in having both.

File: plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_notification_plugincheck_exceededLicense.png
Comment (bx80) [reply plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_notification_plugincheck_exceededLicense.png:1]: The notification mockups have all the links in black.
@Javi-Ormaechea can I confirm that the notification links should be the same colour as the rest of the notification text or do we want to have them in blue to match the rest of the application?

File: plugins/Login/tests/UI/expected-screenshots/Login_bruteforcelog_noentries.png
Comment (bx80) [reply plugins/Login/tests/UI/expected-screenshots/Login_bruteforcelog_noentries.png:1]: I consolidated icon-user-platform and icon-admin-platform as they were the same, that should be fixed now.

File: plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_paid_plugin_details_exceeded_license_superuser.png
Comment (bx80) [reply plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_paid_plugin_details_exceeded_license_superuser.png:1]: I've updated the marketplace .metadata CSS class to make them blue.

File: plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_superuser_enable_plugins_admin_with_multiserver_enabled.png
Comment (bx80) [reply plugins/Marketplace/tests/UI/expected-screenshots/Marketplace_superuser_enable_plugins_admin_with_multiserver_enabled.png:1]: Should be fixed now.

File: plugins/Morpheus/Controller.php (hunk 1 above)
Comment (bx80) [reply plugins/Morpheus/Controller.php:140]: As far as I can see that's the only remaining difference, @michalkleiner has already created a ticket to deprecated one in favour of the other :+1:

File: plugins/Overlay/tests/UI/expected-screenshots/Overlay_loaded_from_actions.png
Comment (bx80) [reply plugins/Overlay/tests/UI/expected-screenshots/Overlay_loaded_from_actions.png:1]: That particular test is taking a screenshot of a popup which is defaulting to 800x600, all the rest of the screenshots are 1350x768. I can't see any obvious way to resize the popup in the UI test but I don't think it's related to this change since the previous screenshot is also cut off.

File: plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_anonymizelogdata_anonymizeip_and_visit_column_cancelled.png
Comment (bx80) [reply plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_anonymizelogdata_anonymizeip_and_visit_column_cancelled.png:1]: They certainly should be consistent :+1: There is an additional ticket #20475 created to unify the drop down design (which I'll work on as soon as this PR is complete)

File: plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_users_opt_out_default.png
Comment (bx80) [reply plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_users_opt_out_default.png:1]: Ouch, I was responsible for that monstrosity! I've removed the text colour override and it's now a standard readable info notification. :sunglasses:

File: plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated.png
Comment (bx80) [reply plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated.png:1]: I've rebased this PR on 5.x-dev, so hopefully that will fix it on the next run.

File: plugins/TwoFactorAuth/tests/UI/expected-screenshots/TwoFactorAuthUsersManager_edit_with_2fa_reset_confirmed.png
Comment (bx80) [reply plugins/TwoFactorAuth/tests/UI/expected-screenshots/TwoFactorAuthUsersManager_edit_with_2fa_reset_confirmed.png:1]: I've moved `icon-arrow-left-2` and `icon-arrow-right-2` to the deprecated section of the font selection and updated user manager to use `icon-arrow-left`. Just trying to be cautious with deleting them entirely just yet in case there is a plugin somewhere using them.

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png
Comment (bx80) [reply plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png:1]: I'll revert this in the next commit :+1:

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_bounce_rate.png
Comment (bx80) [reply plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_bounce_rate.png:1]: I'll revert this in the next commit :+1:

File: plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_permissions_bulk_access_set.png
Comment (bx80) [reply plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_permissions_bulk_access_set.png:1]: I've seen that one a few times, I think it's a scrolling thing, will revert.

File: plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_tab.png
Comment (bx80) [reply plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_tab.png:1]: It's a bit small and not being a solid colour it's harder to see. I've slightly increased the size of the icon just for this usage. 
image

File: tests/UI/expected-screenshots/UIIntegrationTest_actions_pages_tooltip_help.png
Comment (bx80) [reply tests/UI/expected-screenshots/UIIntegrationTest_actions_pages_tooltip_help.png:1]: The bottom right italic text might not work well in an info notification. Maybe there are other improvements that could be done to the report info help box at the same time so it might be worth getting some UX input on another ticket.

File: tests/UI/expected-screenshots/UIIntegrationTest_category_help.png
Comment (bx80) [reply tests/UI/expected-screenshots/UIIntegrationTest_category_help.png:1]: Yes that looks awful, help notifications didn't get covered in the mockups, the style is a derivative of the info notification. For now I've removed the dark blue background from the style but retained the different icon and it looks a lot better. @Javi-Ormaechea can review when the next round of UI tests have completed.

image

**Comment by bx80** [comment]
I've committed the following changes which should address all issues raised so far:

- Temporarily updated submodule references for easier testing :warning: MUST REVERT BEFORE MERGING :warning:
- Updated the 25 outstanding icons used within core, both individual .svg files and updated font files.
- `icon-done` now uses the same thin tick icon as `icon-ok`.
- `icon-plus-square` and `icon-minus-square` now have squares again.
- New icons `icon-add1` and `icon-remove` have been added for where we need simple + / -.
- New icons `icon-chevron-right` and `icon-chevron-left` have been added.
- Updated mobile down menu, dashboard settings, site selector, fieldExandableSelect and reporting menu controls to use a chevron down arrows as before (this will be tidied up further in [#20475](https://github.com/matomo-org/matomo/issues/20475)
- The Morpheus UI demo icons section has been tidied to only show icons with the new style and some missing icons added.
- Fixed the link colour on the marketplace metadata sidebar.
- Fixed the colour on the marketplace plugin tile more link.
- Fixed the ugly privacy manager opt-out non-standard notification colours.
- Increased the size of the warning icon on user permissions menu so it's more visible with outline icons.
- Adjusted the help notification background colour so it matches an info notification, just with a different icon.
- Reverted three UI test screenshot from a previous commit which were not related to style changes.
- Rebased on 5.x-dev

File: plugins/Marketplace/stylesheets/marketplace.less [hunk 2]
```diff
@@ -98,7 +98,7 @@
             padding-bottom: 0;
             margin-bottom: 10px;
             .more {
-                color: @theme-color-text;
+                color: #1976d2;
```
Comment (sgiehl) [inline plugins/Marketplace/stylesheets/marketplace.less]: Should be better to use the variable.
```suggestion
                color: @theme-color-link;
```

File: plugins/Marketplace/stylesheets/marketplace.less [hunk 3]
```diff
@@ -111,6 +111,11 @@
             font-size: 95%;
             margin: 15px 15px 10px;
             list-style: none;
+            a {
+                color: #1976d2;
```
Comment (sgiehl) [inline plugins/Marketplace/stylesheets/marketplace.less]:
```suggestion
                color: @theme-color-link;
```

File: plugins/Dashboard/tests/UI/expected-screenshots/DashboardManager_create_new.png
Comment (sgiehl) [inline plugins/Dashboard/tests/UI/expected-screenshots/DashboardManager_create_new.png:1]: The placement of the dropdown icons when displayed right looks not nice. See the dashboard selector in main menu

File: plugins/Live/tests/UI/expected-screenshots/Live_visitor_profile_action_tooltip.png
Comment (sgiehl) [inline plugins/Live/tests/UI/expected-screenshots/Live_visitor_profile_action_tooltip.png:1]: This one was incorrectly updated

File: plugins/Login/tests/UI/expected-screenshots/Login_password_reset_complete.png
Comment (sgiehl) [inline plugins/Login/tests/UI/expected-screenshots/Login_password_reset_complete.png:1]: Might not be needed for this PR, but shouldn't this notification message maybe also use one of the new styles? Maybe the success style would match here, as it's shown after the password was successfully resetted?

File: plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_autocomplete_capitalized.png
Comment (sgiehl) [inline plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_autocomplete_capitalized.png:1]: this one has incorrect changed content

File: plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_autocomplete_lowercase.png
Comment (sgiehl) [inline plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_autocomplete_lowercase.png:1]: this one, too

File: plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_autocomplete_uppercase.png
Comment (sgiehl) [inline plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_autocomplete_uppercase.png:1]: and here...

File: plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated.png
Comment (sgiehl) [inline plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated.png:1]: also incorrectly updated

File: plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated_details.png
Comment (sgiehl) [inline plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_updated_details.png:1]: here as well

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png
Comment (sgiehl) [inline plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png:1]: Tooltip is now missing

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_bounce_rate.png
Comment (sgiehl) [inline plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_bounce_rate.png:1]: Tooltip missing

File: tests/UI/expected-screenshots/Menus_admin_loaded.png
Comment (sgiehl) [inline tests/UI/expected-screenshots/Menus_admin_loaded.png:1]: platform icon is still missing here

**Review by sgiehl — CHANGES_REQUESTED** [review]
Had another look through the screenshots. Added a couple of comments, where screenshot changed the content unexpected, or some stuff wasn't correct.

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png
Comment (bx80) [reply plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_avg_time_on_site.png:1]: Should now be fixed.

File: plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_bounce_rate.png
Comment (bx80) [reply plugins/UserCountryMap/tests/UI/expected-screenshots/VisitorMap_bounce_rate.png:1]: Should now be fixed.

File: tests/UI/expected-screenshots/Menus_admin_loaded.png
Comment (bx80) [reply tests/UI/expected-screenshots/Menus_admin_loaded.png:1]: Looks like the previous fix got lost in the last font update somehow, it should be sorted now.

File: plugins/Login/tests/UI/expected-screenshots/Login_password_reset_complete.png
Comment (bx80) [reply plugins/Login/tests/UI/expected-screenshots/Login_password_reset_complete.png:1]: The same infoMessage box is also used to notify that the password link has been sent, so I've updated it to use the info style since that will work for both.

File: plugins/Dashboard/tests/UI/expected-screenshots/DashboardManager_create_new.png
Comment (bx80) [reply plugins/Dashboard/tests/UI/expected-screenshots/DashboardManager_create_new.png:1]: It seems okay to me, is this what you mean ? 
image
There is another ticket to be done right after this one where dropdown alignment and icons will be tidied up, so maybe any tweaks to this should be done there?

File: plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_autocomplete_capitalized.png
Comment (bx80) [reply plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_autocomplete_capitalized.png:1]: The segment selector tests are broken on other [builds](https://builds-artifacts.matomo.org/matomo-org/matomo/DEV-16595-no-data-screen-translations-review/5793646777/) too, so it look like either something has recently changed in the test or it is randomly failing.

File: plugins/Login/tests/UI/expected-screenshots/Login_bruteforcelog_noentries.png
Comment (bx80) [reply plugins/Login/tests/UI/expected-screenshots/Login_bruteforcelog_noentries.png:1]: I've confirmed this is fixed on the latest tests.

File: plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_tab.png
Comment (bx80) [reply plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_tab.png:1]: Confirmed the larger icon is now shown on the latest tests.

File: tests/UI/expected-screenshots/UIIntegrationTest_category_help.png
Comment (bx80) [reply tests/UI/expected-screenshots/UIIntegrationTest_category_help.png:1]: I can confirm this is showing the updated info style on the latest test screenshots.

File: plugins/Login/tests/UI/expected-screenshots/Login_password_reset_complete.png
Comment (bx80) [reply plugins/Login/tests/UI/expected-screenshots/Login_password_reset_complete.png:1]: I can confirm the latest test screenshots now show use the info style

File: tests/UI/expected-screenshots/Menus_admin_loaded.png
Comment (bx80) [reply tests/UI/expected-screenshots/Menus_admin_loaded.png:1]: Confirmed fixed in the latest tests.

File: plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_users_opt_out_default.png
Comment (bx80) [reply plugins/PrivacyManager/tests/UI/expected-screenshots/PrivacyManager_users_opt_out_default.png:1]: Confirmed fixed in the latest test run.
